### DISCUSSION TITLE
Version 1.3

### DISCUSSION THREAD

**Discussion by LeagueOfPoro** [LeagueOfPoro/CapsuleFarmerEvolved#120] [Announcements]
## How to install
Check the [Quickstart Guide for Windows](https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/wiki/Quickstart-Guide-(Windows))

## How to upgrade
Replace the `.exe` file in your current installation with the single **exe** file in the releases here. (or from the ZIP)

## What's Changed
- Major stability improvements
- Better resiliency against user error
- Reduce network load with multiple accounts
- Display where the last drop came from
- Improved documentation

## Changelog
* style: Visual Update confighelper.html by @vitorsammarco in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/71
* refactor!: BREAKING - Refactoring paths, folders and files by @auroraisluna in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/72
* fix: beststreams.txt not found by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/78
* fix: Removed instancing of log helper. by @hypixus in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/88
* docs: Banner + readme changes by @Penguax in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/83
* fix: Update Dockerfile by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/89
* feat: Remove dummy accounts from config by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/91
* feat: Automatically restart threads with increasing delay by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/90
* fix: docker table display by @DjordyKoert in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/56
* feat: Dynamically loading bestStreams from Url in config by @wiflow in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/87
* fix: Possible infinite loop in session refresh. by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/94
* fix: Properly close all connections by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/97
* fix: Correctly catch RateLimitException by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/109
* fix: Improve resiliency against 403 from watch heartbeat by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/114
* feat: change log handler to RotatingFileHandler by @s-Sizz in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/111
* feat: Added time until next match, when there are no live matches. by @wiflow in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/99
* feat: Move common tasks to a separate thread by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/115
* feat: Refactor the use of StatusCodeException by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/116
* feat: Added last drop league, to the last drop field by @wiflow in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/96
* feat: Added Drop and 2FA Notifications and Toasts [BREAKING] by @Mazen-Ghaleb in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/110
* fix: Fix failing watch requests detection by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/118
* Revert "feat: Added Drop and 2FA Notifications and Toasts [BREAKING]" by @LeagueOfPoro in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/119

## New Contributors
* @vitorsammarco made their first contribution in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/71
* @hypixus made their first contribution in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/88
* @wiflow made their first contribution in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/87
* @s-Sizz made their first contribution in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/111
* @Mazen-Ghaleb made their first contribution in https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/pull/110

**Full Changelog**: https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/compare/v1.2...v1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/releases/tag/v1.3'>Version 1.3</a>.</em>

**Comment by TimMorningstar** [comment]
Need a help, wanted to run it on MacOs but failed to do so. Ventura 13.1

**Comment by PetarWho** [comment]
Need help, when using 2 accounts, the first logs in and goes live but the second one fails to login. Always giving ERROR - restart at .....
And just cant log in no matter how many attempts pass. The password and name are correct. I don't know what happened. worked flawlessly 3 days ago

**Reply by Mahruky** [reply]
try disabling 2fa on your second account .

**Reply by PetarWho** [reply]
Neither account has 2fa

**Reply by PetarWho** [reply]
Found the solution. Just change the name of the second account... It fixed my problem

**Comment by ThePlagueDoctorOftheyear1619** [comment]
is this bannable?, I trust the code but I just worry about losing my account forever/making a new one

**Reply by alepouna** [reply]
FAQ

**Comment by bojo509** [comment]
What is the difference between red colored and green colored matches in the "Live section"?
